content/docs/1.0.0/tutorial/deploy-your-microservice.md

```diff
@@ -86,34 +85,114 @@ With the microservice UUID from the last step, let's change our routes so that o
 
 First, let's remove the old route from the Sensors to the REST API. We need to retrieve the microservice UUIDs for Sensors and the REST API:
 
-```sh
-iofog-controller microservice list
+```bash
+docker exec -ti iofog-controller iofog-controller microservice list
 ```
 
-After finding those two UUIDs in the list, provide the Sensors UUID and API UUID separated by a semicolon to `microservice route-remove --route <source_uuid>:<dest_uuid>`:
+Example output can look like this. Note that many attributes are not shown in this output, but we can clearly see all the UUIDs and routes.
+```json
+{
+  "microservices": [
+    {
+      "name": "Sensors",
+      "uuid": "PDRRQcbD6DVZJy9QBJQB7JyzH7RgLJCb",
+      "catalogItemId": 102,
+      "iofogUuid": "vH6vfNzLwHK483CwHRJyyPZtNZ2m46zC",
+      "routes": [
+        "NZp8HZ7xpztPyC4dpRQx4w3Jd8x9jNF3"
+      ]
+    },
+    {
+      "name": "Rest API",
+      "uuid": "NZp8HZ7xpztPyC4dpRQx4w3Jd8x9jNF3",
+      "catalogItemId": 103,
+      "iofogUuid": "vH6vfNzLwHK483CwHRJyyPZtNZ2m46zC",
+    },
+    {
+      "name": "Freeboard",
+      "uuid": "BrKHZf9PTcT6yjKcrpnRVBcYPFqxqXxb",
+      "catalogItemId": 104,
+      "iofogUuid": "vH6vfNzLwHK483CwHRJyyPZtNZ2m46zC",
+    },
+    {
+      "name": "Moving Average",
+      "uuid": "ydmtBFxJhxvVgLKfM2qLPj4KtcVgdg23",
+      "config": "{ \"maxWindowSize\": 10 }",
+      "catalogItemId": 105,
+      "iofogUuid": "vH6vfNzLwHK483CwHRJyyPZtNZ2m46zC",
+    }
+  ]
+}
+```
 
-```sh
-# Note the semicolon between the two UUIDs!
-iofog-controller microservice route-remove \
-  --route <sensors_uuid>:<api_uuid>
+After finding those two UUIDs in the list, provide the Sensors UUID and API UUID separated by a semicolon.
+```bash
+docker exec -ti iofog-controller \
+    iofog-controller microservice route-remove \
+        --route PDRRQcbD6DVZJy9QBJQB7JyzH7RgLJCb:NZp8HZ7xpztPyC4dpRQx4w3Jd8x9jNF3
 ```
 
 Now we need to place two new routes: one from the Sensors to Moving Average, and another from Moving Average to the REST API; this places our new microservice in between them.
```

Comment:
Now we need to place two new routes: one from the Sensors to Moving Average~,~ and another from Moving Average to the REST API**.** **The result** places our new microservice in between them.